playground/frontend/lib/modules/examples/components/example_list/category_expansion_panel.dart

```diff
@@ -54,8 +54,8 @@ class CategoryExpansionPanel extends StatelessWidget {
             children: [
               Transform.rotate(
                 angle: -math.pi * animationValue,
-                child: const Icon(Icons.keyboard_arrow_down),
```

Comment:
Linter now recommends to order 'child' property last as it tends to be the largest one.